### DISCUSSION TITLE
Make a great speaker.npz file? Leave it here and I'll include it in future updates.

### DISCUSSION THREAD

**Discussion by JonathanFly** [JonathanFly/bark#47] [Ideas]
You don't need the original WAV because that can be regenerated, but if you have the exact original prompt text that would be helpful. Anything details are helpful: a brief description, a link to something you used the speaker.npz file to create, etc. 

Or just drop off a bunch of .npz files and I'll sort it out eventually.

**Comment by C0untFloyd** [comment]
Just a gift from your friendly bark neighbourhood 😃 

[vincentprice.zip](https://github.com/JonathanFly/bark/files/11615524/vincentprice.zip)

Vincent Price cloned with the new method (without prompt), which is working lots better than previously.
Rename the .zip extension to .npz to use it.
While we're at it, did you find a way to build fairseq on Windows?

**Reply by JonathanFly** [reply]
Thanks. 

I found that @Sharrnah built it. 
pip install fairseq@https://github.com/Sharrnah/fairseq/releases/download/v0.12.4/fairseq-0.12.4-cp310-cp310-win_amd64.whl

**Comment by C0untFloyd** [comment]
Oooh this is cool, thank you! Finally no more colab for me, easy high quality voice cloning possible.
Or as D. Attenborough might say
 
[final_22-01-26_s4127791140.webm](https://github.com/JonathanFly/bark/assets/131583554/62cd7795-7f7b-4e8b-a1cc-1fba7cc27afc)

Here's the speaker:
[attenborough.zip](https://github.com/JonathanFly/bark/files/11618515/attenborough.zip)
